routes/questions.go

```diff
@@ -1,80 +0,0 @@
-package routes
-
-import (
-	"encoding/json"
-	"io/ioutil"
-	"net/http"
-
-	"github.com/personal_trivia/repo"
-
-	"github.com/gorilla/mux"
-)
-
-type Question struct {
```

Comment:
I dont think you should separate your models since their pretty much identical. I would use both tags on the struct.